dsp/modules/hf_client.py

```diff
@@ -212,13 +212,19 @@ def __init__(self, model, **kwargs):
         self.api_base = os.getenv("TOGETHER_API_BASE")
         self.token = os.getenv("TOGETHER_API_KEY")
         self.model = model
+
+        self.use_inst_template = False
+        if "inst" in self.model.lower() or "instruct" in model.lower():
+            self.use_inst_template = True
+
         self.kwargs = {
             "temperature": 0.0,
             "max_tokens": 512,
             "top_p": 1,
             "top_k": 20,
             "repetition_penalty": 1,
             "n": 1,
+            "stop": ["\n\n", "---", "[/INST]"],
```

Comment:
Is this the final thing we decided or did you want to try the \n\n---?

Reply:
Hi @okhat 

I was testing locally with ` \n\n---` yesterday.
Now I have updated the PR, PTAL.

Thank you.